whereby/sdk
# Core: add notification for client unable to join full room

-------

### Description
Lets the host know if another user tries to join a full room
**Summary:**



**Related Issue:**
https://linear.app/whereby/issue/COB-972/add-an-sdk-host-notification-when-someone-tries-to-join-a-full-room



### Testing
1. yarn build
1. yarn dev:sample-app
1. go to the sample app page and join a P2P room as a host
1. join the same room with 3 more clients in other browsers
1. try to join from a fifth browser
1. see the notification in the sample app 


### Screenshots/GIFs (if applicable)
<img width="1050" alt="image" src="https://github.com/user-attachments/assets/ffd48d51-1c0f-4764-a9bc-2f1d2c6b435a">



### Checklist

        under the terms of the MIT license.

### Additional Information
